jetty-hazelcast/src/main/config/modules/session-store-hazelcast-embedded.mod

```diff
@@ -31,7 +31,6 @@ http://www.apache.org/licenses/LICENSE-2.0.html
 [ini-template]
 jetty.session.hazelcast.mapName=jetty-distributed-session-map
 jetty.session.hazelcast.hazelcastInstanceName=JETTY_DISTRIBUTED_SESSION_INSTANCE
-#jetty.session.hazelcast.configurationLocation=
```

Comment:
it must be still possible to configure hazelcast in embedded mode so I would prefer leave it but it's not in the xml and it should be maybe you can add it as well?